QuoteVerification/dcap_quoteverify/sgx_dcap_pcs_com.cpp

```diff
@@ -238,7 +271,7 @@ quote3_error_t sgx_dcap_free_qve_identity(
  **/
 quote3_error_t tdx_dcap_retrieve_verification_collateral(
     const char *fmspc,
-    uint16_t fmspc_size,
+    __attribute__((unused))uint16_t fmspc_size,
```

Comment:
Is this code tested on Windows? I think we also need to support Windows build. While this key word is Linux specific.

Reply:
Sorry, I forgot to delete it. I did it for compiling without warnnings because there was no TDX support at that time. Actually, I'm not very familiar with Windows so that It just supports Linux build by now, do I need to implement it to support Windows build ?

Reply:
I'm OK not to enable the static libs generation on Windows if this is just a Linux requirement.  But we need to ensure the PR doesn't impact current Windows build and execution.

Reply:
OK, I have fixed it, please check.